document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AccountsHandlerImpl.java

```diff
@@ -83,5 +88,43 @@ private AccountType getAccountType(Accounts accountsData) throws HandlerExceptio
     private String getAccountLink(Accounts accounts, AccountType accountsType) {
         return accounts.getLinks().get(accountsType.getResourceKey());
     }
-    
+
+    /**
+     * Creates the 'data' string in {@link DocumentInfoResponse}.
+     * @param transaction the transaction data
+     * @param abridgedAccountData the abridged accounts data
+     * @return data string in {@link DocumentInfoResponse}
+     */
+    private String createDocumentInfoResponseData(Transaction transaction, AbridgedAccountsApi abridgedAccountData) {
+        JSONObject abridgedAccountJSON = new JSONObject(abridgedAccountData);
+        JSONObject abridgedAccount = new JSONObject();
+        abridgedAccount.put("abridged_account", abridgedAccountJSON);
+        abridgedAccount.put("company_name", transaction.getCompanyName());
+        abridgedAccount.put("company_number", transaction.getCompanyNumber());
+        return abridgedAccount.toString();
+    }
+
+    /**
+     * Creates the {@link DocumentInfoResponse} object
+     * @param transaction transaction data
+     * @param accountType account type
+     * @param abridgedAccountData abridged account data
+     * @return {@link DocumentInfoResponse} object
+     */
+    private DocumentInfoResponse createResponse(Transaction transaction, AccountType accountType, AbridgedAccountsApi abridgedAccountData) {
+        DocumentInfoResponse documentInfoResponse = new DocumentInfoResponse();
+        documentInfoResponse.setData(createDocumentInfoResponseData(transaction, abridgedAccountData));
+        documentInfoResponse.setAssetId(accountType.getAssetId());
+        documentInfoResponse.setTemplateName(accountType.getTemplateName());
+        documentInfoResponse.setLocation(createLocationString(accountType));
+        return documentInfoResponse;
+    }
+
+    private String createLocationString(AccountType accountType) {
+        return String.format("%s/%s/%s", getBucketName(), accountType.getAssetId(), accountType.getUniqueFileName());
```

Comment:
When is the prefix "s3://" being appended to the location?

Reply:
So this uses the `DOC_GEN_ACC_BUCKET_NAME` variable which will end up being: `DOC_GEN_ACC_BUCKET_NAME` = $STORAGE_TYPE://$DOCUMENT_RENDER_SERVICE_HOST`

Reply:
Have you raised a story for updating the configs when we make the switch to this new repo?

Reply:
What would you recommend calling it? The story has been created as SFA-725. Will obviously change when we agree on a name for it.

Reply:
As discussed offline, we will be using the `bucketName` in doc-gen-api and we will only return the path which does not need the `bucketName` so this PR will not need to worry about it.